ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -121,55 +167,96 @@ export default defineComponent({
   components: {
     FolioNumber
   },
+  props: {
+    isReviewMode: { default: false }
```

Comment:
we usually have to load this into a computed value as the props values sometimes don't register the update... not sure if you will find this to be true :)

Reply:
Will look into this, although i didn't notice this in development. It could be perhaps since it's re-mounted when we mount this component in Review Mode. 
But yea, thanks for the heads up here. Will look at this! 

Specifically, were you finding that the components were not being reactive to props in general?